x-pack/platform/plugins/shared/streams_app/public/hooks/use_dashboards_api.ts

```diff
@@ -38,15 +34,13 @@ export const useDashboardsApi = (name?: string) => {
           },
         },
       });
+      refreshDefinition();
```

Comment:
💬 happy to move that out and let the consumer of the hook decides whether or not they should refresh.